composer.json

```diff
@@ -2,25 +2,25 @@
   "name": "nuvoleweb/robo-config",
   "description": "CI/developer-friendly configuration processor for Robo.",
   "type": "robo-tasks",
-  "license": "GPL-2.0",
+  "license": "GPL-2.0-or-later",
   "authors": [
     {
       "name": "Antonio De Marco",
       "email": "antonio@nuvole.org"
     }
   ],
   "require": {
-    "consolidation/robo": "^2.1"
+    "consolidation/robo": "^3.0"
   },
   "require-dev": {
     "drupal/coder": "^8",
     "phpunit/phpunit": "^7",
-    "phpro/grumphp": "0.14.0"
+    "phpro/grumphp": "^0.22"
```

Comment:
```suggestion
    "phpro/grumphp": "^1.2"
```

Reply:
I just wanted to make it work with `"consolidation/robo": "^3.0"`,
with as little modification as possible.
I upgraded grumphp to 0.22 from 0.14, because 0.14 throws error messages, with PHP 7.4.

Deprecated: strpos(): Non-string needles will be interpreted as strings in the future. Use an explicit chr() call to preserve the current behavior in /.../vendor/phpro/grumphp/src/Locator/ConfigurationFile.php on line 75